packages/frontend-2/lib/auth/composables/onboarding.ts

```diff
@@ -1,7 +1,12 @@
 import { useApolloClient } from '@vue/apollo-composable'
 import { useMixpanel } from '~~/lib/core/composables/mp'
 import { UnsupportedEnvironmentError } from '~~/lib/core/errors/base'
-import type { OnboardingState } from '~~/lib/auth/helpers/onboarding'
+import type {
```

Comment:
Maybe these types should be in the defined here instead, so both BE and FE can validate it's actually part of the enum rather than just a random string. It will probably require you to move these consts to shared

```
input OnboardingCompletionInput {
  role: String
  plans: [String!]
  source: String
}
```